features/page_objects/back_office/new/user_accept_invite_page.rb

```diff
@@ -0,0 +1,10 @@
+require_relative "sections/govuk_banner.rb"
+
+class UserAcceptInvitePage < SitePrism::Page
+  section(:govuk_banner, GovukBanner, GovukBanner::SELECTOR)
+
+  element :password_field, "#user_password"
+  element :confirm_password_field, "#user_password_confirmation"
+
+  element :submit_field, 'input[type="submit"]'
```

Comment:
Another picky one. I usually write the quotes the other way round, e.g. `"input[type='submit']"` If this passes rubocop I'm happy, but would prefer the other way round for consistency.

Reply:
👍